src/tests/ftest/io/daos_perf.py

```diff
@@ -107,7 +102,7 @@ def runner(self, test_path):
                              "Failed", "failed"]
 
             for line in searchfile:
-                for i in xrange(len(error_message)):
+                for i in range(len(error_message)):
```

Comment:
(pylint-consider-using-enumerate) Consider using enumerate instead of iterating with range and len